gcc/rust/parse/rust-parse-impl.h

```diff
@@ -1427,6 +1434,40 @@ Parser<ManagedTokenSource>::parse_vis_item (AST::AttrVec outer_attrs)
   return nullptr;
 }
 
+template <typename ManagedTokenSource>
+std::unique_ptr<AST::Function>
+Parser<ManagedTokenSource>::parse_async_item (AST::Visibility vis,
+					      AST::AttrVec outer_attrs)
+{
+  const_TokenPtr t = lexer.peek_token ();
+  if (Session::get_instance ().options.get_edition ()
+      == CompileOptions::Edition::E2015)
+    {
+      add_error (Error (Error::Kind::Err, t->get_locus (), ErrorCode::E0670,
+			" %<`async fn`%> is not permitted in Rust 2015"));
```

Comment:
`%<` already puts a ` in your output text, you do not need to keep those `

Reply:
Done :D